charts/kserve/templates/configmap.yaml

```diff
@@ -38,7 +38,7 @@ data:
       limits:
         cpu: "2"
         memory: "512Mi"
-    serviceAccountName: modelmesh
+    serviceAccountName: ""
```

Comment:
is this service account used by modelmesh controller, any reason to be empty?

Reply:
This was the service account used by runtime pods and was mainly for internal usage for passing in private registry  secrets to the runtime. Not needed by default.